### PR TITLE
Icon Redesign

### PR DESCRIPTION
**Pull request by Fatih20**
The clock at the bottom in my opinion wasn't compliant by the [eOS HIG](https://elementary.io/docs/human-interface-guidelines#shadows) regarding the pictogram in the form of a clock at the corner. Also, the highlight outline is spilling over outside so I mask it to keep that from happening.

### CODE REVIEW

**Comment by lainsce** [comment]
Why some icons have a black transparent background and others don't? Let the highlight outline bleed.
Other than that, I don't like the new pictogram style. It's a clock printed in the paper, not a sunken clock.

**Comment by lainsce** [comment]
Since there was no reply to my comment above, I'll close this.

**Comment by Fatih20** [comment]
Sorry, I'm on a different timezone. I guess you're right that a sunken clock doesn't make sense on a supposed calendar. But, it also looks weird that the clock is flat while the highlights and shadows on the calendar's header make it look skeuomorphic. I feel like there are disconnects there. What about and subtly extruded clock? So, it's not a clock written on a calendar, but a clock sticker sticked onto a calendar?

**Comment by lainsce** [comment]
It's a clock printed in the paper, it's like text  on paper, not a sticker nor anything else. :roll_eyes:

**Comment by Fatih20** [comment]
I know that it's a clock printed on a paper, in that case, it should be flat. Problem is, it's flat while being right next to a skeuomorphic looking calendar because of the highlights and shadows. It looks like they're from a different icon set just slapped together.

**Comment by Fatih20** [comment]
In that case, wouldn't it be better to have a time (12.00, 06.00, etc.) written on it and maybe a marker highlighting it instead of a clock. Who draws a clock on a timetable? People write time signals not a clock. People (like me) who sees your icon wouldn't think that it's a clock written on a paper (again, who draws a clock on a timetable). They would think it's just a clock (that's for some reason is flat instead of skeuomorphic like the timetable next to it) next to a timetable. 

So, in summary, IMO you should replace the clock with a time signals highlighted by a marker. Because it's easier to understand and it's easier to convey the use of the apps.

**Comment by lainsce** [comment]
"Who draws a clock on a timetable?"

Have you heard of "show , don't tell"? By putting in a clock, I give the _idea_ of time without having to write a time signal. It's _elementary_. It's minimal. And gets the point across.

**Comment by Fatih20** [comment]
Huh, you're right. Learned something new today.
